lang/cs/Org.Apache.REEF.Utilities/Logging/Logger.cs

```diff
@@ -140,7 +151,7 @@ public bool IsLoggable(Level level)
         /// <param name="args"></param>
         public void Log(Level level, string formatStr, params object[] args)
         {
-            if (CustomLevel >= level)
+            if ((CustomLevel != Level.Unset ? CustomLevel : GlobalLevel) >= level)
```

Comment:
I would replace this expression with a call to `IsLoggable(level)`, and same for line 175. This way we carry the logic of merging the two levels in a single place, which is more convenient for potential future updates.